psi4/share/psi4/basis/def2-qzvp-ri.gbs

```diff
@@ -3,7 +3,7 @@ spherical
 !
 ! Basis sets obtained from http://www.cosmologic.de/basis-sets/basissets.php  03/17/14
 !
-! C. Hättig; PCCP 7, 59 (2005).
+! C. Hattig; PCCP 7, 59 (2005).
 !
```

Comment:
Non unicode H¨attig was evenly divided into `a` and `ae`. I can do a standardization pass at some point.